test/WebJobs.Script.Tests/Eventing/SystemLoggerTests.cs

```diff
@@ -64,13 +64,17 @@ public void Trace_Verbose_LogData_EmitsExpectedEvent()
             string message = "TestMessage";
             string functionInvocationId = Guid.NewGuid().ToString();
             string activityId = Guid.NewGuid().ToString();
+            var scopeState = new Dictionary<string, object>
+            {
+                [ScriptConstants.LogPropertyFunctionInvocationIdKey] = functionInvocationId
+            };
+            DictionaryLoggerScope.Push(scopeState);
 
             _mockEventGenerator.Setup(p => p.LogFunctionTraceEvent(LogLevel.Debug, _subscriptionId, _websiteName, _functionName, eventName, _category, details, message, string.Empty, string.Empty, functionInvocationId, _hostInstanceId, activityId));
 
             var logData = new Dictionary<string, object>
             {
                 ["message"] = message,
```

Comment:
Even though it's a test, for the purpose of "best practices", can you remove this `message` property and, instead, call the `Log` method like:

`_logger.Log(LogLevel.Debug, 0, logData, null, (state, ex) => message);`

That's how you can pass a message to a logger and have it be separate from the state, which it looks like you're trying to do here.